### PR TITLE
Add version string, fix tests

### PR DESCRIPTION
**Pull request by OfLoopa**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,6 +62,7 @@ Then run in the current directory
 This will ...
 
 ```yaml
+version: '3.8'
```
Comment (artyomb) [inline README.md:65]: Is 3.8 the latest version?

**Comment by OfLoopa** [comment]
test conversation

**Comment by artyomb** [comment]
> 

Pong

**Review by artyomb — CHANGES_REQUESTED** [review]
Review completed

File: README.md (hunk 1 above)
Comment (OfLoopa) [reply README.md:65]: No

**Review by artyomb — APPROVED** [review]

**Pull request merged**